usaspending_api/awards/v2/filters/view_selector.py

```diff
@@ -206,88 +206,6 @@ def can_use_view(filters, view_name):
     return True
 
```

Comment:
I imagine all of the dictionaries defined in `MATVIEW_SELECTOR` can be removed except for the three listed in `recipient_totals()`.

Reply:
Hesitant to remove them from this PR since those matviews are still around as of right now. Was thinking they might be removed as part of removing the old matviews.

Reply:
Using that logic, Emily could say "I can't remove these matviews, they are still used by the code!" 😆 

Time-permitting, it would be good to be as aggressive with code removal as possible. Shifting priorities might not leave much more time for this

Reply:
Touché. Will remove that in my next commit.

Reply:
I'd agree, seems more in line with removing the matviews, but it should be fine either way